Transactions.md

```diff
@@ -169,3 +169,45 @@ Where
 
 Where
   - `P2WPKH(Local Penalty Public Key)`'s value is `P2WSH(Remote's ToLocalOutput).value - fee`
+
+## Mutual Closing Transaction
```

Comment:
I would prefer this case go above the two non-cooperative closing cases as it is the most common case.

Reply:
I moved above Closing Transaction (Unilateral).